tests/examples/pacer/nef/s3/paeb_3.json

```diff
@@ -4,45 +4,22 @@
   "court_id": "paeb",
   "dockets": [
     {
-      "case_name": "Tri-State Paper, Inc. v. Foodarama Caterers Inc.",
+      "case_name": "Kervince Markenzy Michel",
       "date_filed": null,
       "docket_entries": [
         {
           "date_filed": "2024-02-01",
-          "description": "Adversary Case 2:23-ap-75 Terminated for Statistical Purposes. (R., LastName)",
-          "document_number": null,
-          "document_url": null,
-          "pacer_case_id": "510318",
-          "pacer_doc_id": null,
-          "pacer_magic_num": null,
-          "pacer_seq_no": null,
-          "short_description": ""
+          "description": "Chapter 13 Plan Filed by Kervince Markenzy Michel. (FirstName, LastName)",
+          "document_number": "17",
+          "document_url": "https://ecf.paeb.uscourts.gov/doc1/152032523670?pdf_header=&magic_num=29282863&de_seq_num=78&caseid=510868",
+          "pacer_case_id": "510868",
+          "pacer_doc_id": "152032523670",
+          "pacer_magic_num": "29282863",
+          "pacer_seq_no": "78",
+          "short_description": "Chapter 13 Plan"
```

Comment:
We keep the "Chapter..." string

Email subject:
`Subject:Ch-13 23-13878-pmm Chapter 13 Plan - Kervince Markenzy`

History report
![image](https://github.com/user-attachments/assets/4924785e-aa49-4643-b731-461c599995e2)